### ISSUE TITLE
Font Resource file paths not resolved correctly on JVM target (Mac)

### ISSUE THREAD

**Issue by chris-hatton** [icerockdev/moko-resources#259]
I'm aiming to use MokoResources to load fonts on both **Android** and **Desktop** targets (using Compose/Multiplatform).
Everything works on **Android** 🎉 
On Desktop I see that when resolving a `FontResource` e.g. with expression:
`MR.fonts.Cabin.regular`
...MokoResources internally fails with e.g:
```
Caused by: java.io.FileNotFoundException: Couldn't find font resource at: fonts//Users/chris/Projects/Personal/Redacted/client/presentation/src/materialComposeMain/resources/MR/fonts/Cabin-Italic-Regular.ttf.ttf
```
It looks as though Moko code has **incorrectly used a full absolute path** instead of just the filename, when building the path.

I was executing on Mac, as part of **Compose for Desktop** immediate `run` task e.g. with the Gradle command:
`gradle :client:app:desktop:run`

**Comment by chris-hatton** [comment]
It looks like [this line](https://github.com/icerockdev/moko-resources/blob/e726ae27355d3fe28d69976806bfde9d427f0b8d/resources-generator/src/main/kotlin/dev/icerock/gradle/generator/jvm/JvmFontsGenerator.kt#L24) in `JvmFontsGenerator` should not be implicitly using the `toString` method of `File` and should instead use something like `"$FONTS_DIR/${fileSpec.file.name}"` similarly to `FilesGenerator`.

**Comment by matt-ramotar** [comment]
Hey Team - Also hitting this

**Comment by Alex009** [comment]
will be available in 0.17.4
